src/Bicep.LangServer/CompilationManager/ICompilationManager.cs

```diff
@@ -4,9 +4,9 @@
 using OmniSharp.Extensions.LanguageServer.Protocol;
 using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 
+
 namespace Bicep.LanguageServer.CompilationManager
-{
-    public interface ICompilationManager
+{    public interface ICompilationManager<T>
```

Comment:
This is not a big deal, but the formatting on this file is a bit different from the rest of the repo. Reading code is easier when there's a uniform style in use, so you might want to make this file look a little more like the rest for readability's sake.

Reply:
If you're using VSCode, you can use "Format Document" in the command palette (Ctrl+Shift+P) to auto-fix formatting.